src/ui/public/kuery/lib/kuery.peg

```diff
@@ -56,12 +56,32 @@ Term
   }
 
 
-integer "integer"
-  = digits:[0-9]+ { return parseInt(digits.join(''), 10); }
+ /* ----- Core types ----- */
+
+literal "literal"
```

Comment:
I thought about this some too, it's an annoying problem. I'd love to force quotes around everything to remove ambiguity, but I think it's unintuitive to a beginner who just wants to start typing search terms into the query bar. For now I've just followed Timelion's lead.